src/Controller/Admin/ActivityController.php

```diff
@@ -104,13 +104,19 @@ public function groupAction(Group $group): Response
     public function newAction(Request $request, GroupRepository $groupRepo): Response
     {
         $activity = new Activity();
+        $price = new PriceOption();
+        $activity->AddOption($price);
```

Comment:
`AddOption` is called here, but I'm assuming you mean `addOption`? Please correct this